modules/tensor_mechanics/doc/content/source/materials/lagrangian/ComputeLagrangianElasticEngineeringStress.md

```diff
@@ -0,0 +1,36 @@
+# ComputeLagrangianElasticEngineeringStress
```

Comment:
Is 'Engineering' in this name meaningful/helpful? If not, maybe drop it.

Reply:
It's meaningful in the sense that you are providing the "small strain" linear elasticity formulation and letting it be rotated/updated by the objective stress formulation.  So the thing you're providing is the "EngineeringStress".  There are other (hyperelastic) ways of doing elasticity.

I guess we could say "ObjectiveStress" like the superclass, but I don't like that as you're not providing the objective stress (you're providing the small stress).  You could say "SmallStress" but I have some recollection from sometime in the past in the huge thread above that we switched from "Small" to "Engineering"

You could even say "LinearElasticStress" which as long as you interpret "linear" to mean both material formulation and kinematics.  This would probably be more familiar to people.

I'll be bold and go with `ComputeLagrangianLinearElasticStress`

Reply:
Yeah, I like that new name better, thanks!